autorun.brs

```diff
@@ -275,8 +276,6 @@ Sub EnterEventLoop()
       else
         DebugLog("BS: Unknown eventData: " + type(eventData))
       endif
-    else if type(ev) = "roGpioButton" then
-      if ev.GetInt() = 12 then stop
```

Comment:
wow, what did it do?

Reply:
not sure, but it caused script error once network is disconnected